store/src/main/java/org/apache/rocketmq/store/rocksdb/RocksDBOptionsFactory.java

```diff
@@ -71,7 +71,7 @@ public static ColumnFamilyOptions createCQCFOptions(final MessageStore messageSt
                 setTableFormatConfig(blockBasedTableConfig).
                 setMemTableConfig(new SkipListMemTableConfig()).
                 setCompressionType(CompressionType.LZ4_COMPRESSION).
-                setBottommostCompressionType(CompressionType.ZSTD_COMPRESSION).
+                setBottommostCompressionType(CompressionType.LZ4_COMPRESSION).
```

Comment:
The default parameters remain unchanged
 may be parameterized is better

Reply:
OK, let's parameterize this option; However, I prefer to let it default to an option that makes more sense instead of keeping it unchanged. Will this preference make sense for you? @fuyou001

Reply:
We have used ZSTD_COMPRESSION

Reply:
All right, let's make it configurable in the next PR